includes/admin/load.php

```diff
@@ -292,7 +292,7 @@ function perflab_install_activate_plugin_callback() {
 
 	$result = activate_plugin( $plugin_basename );
 	if ( is_wp_error( $result ) ) {
-		wp_die( esc_html( $result->get_error_messages() ) );
+		wp_die( esc_html( implode( ' ', $result->get_error_messages() ) ) );
```

Comment:
Alternatively, it's unlikely there is more than one error being returned:

```suggestion
		wp_die( esc_html( $result->get_error_message() ) );
```

Reply:
I think we can go with this suggestion, as it's simpler and I agree it's unlikely there's more than one message.